// angular2-chartjs/dist/chart.component.ngfactory.js
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties,missingOverride}
 */
/* tslint:disable */
import * as import0 from '@angular/core';
import * as import1 from 'angular2-chartjs/dist/chart.component';
var styles_ChartComponent = ['[_nghost-%COMP%] { display: block; }'];
export var RenderType_ChartComponent = import0.ɵcrt({
    encapsulation: 0,
    styles: styles_ChartComponent,
    data: {}
});
export function View_ChartComponent_0(l) {
    return import0.ɵvid(0, [(l()(), import0.ɵeld(0, null, null, 0, 'canvas', [], null, null, null, null, null))], null, null);
}
function View_ChartComponent_Host_0(l) {
    return import0.ɵvid(0, [
        (l()(), import0.ɵeld(0, null, null, 1, 'chart', [], null, null, null, View_ChartComponent_0, RenderType_ChartComponent)),
        import0.ɵdid(638976, null, 0, import1.ChartComponent, [import0.ElementRef], null, null)
    ], function (ck, v) {
        ck(v, 1, 0);
    }, null);
}
export var ChartComponentNgFactory = import0.ɵccf('chart', import1.ChartComponent, View_ChartComponent_Host_0, {
    type: 'type',
    data: 'data',
    options: 'options'
}, {}, []);
