// angular2-chartjs/dist/chart.component.ngfactory.ts
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties,missingOverride}
 */
 /* tslint:disable */


import * as import0 from '@angular/core';
import * as import1 from 'angular2-chartjs/dist/chart.component';
const styles_ChartComponent:any[] = ['[_nghost-%COMP%] { display: block; }'];
export const RenderType_ChartComponent:import0.RendererType2 = import0.ɵcrt({
  encapsulation: 0,
  styles: styles_ChartComponent,
  data: {}
}
);
export function View_ChartComponent_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[(l()(),import0.ɵeld(0,(null as any),(null as any),0,'canvas',([] as any[]),(null as any),(null as any),(null as any),(null as any),(null as any)))],(null as any),(null as any));
}
function View_ChartComponent_Host_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
    (l()(),import0.ɵeld(0,(null as any),(null as any),1,'chart',([] as any[]),(null as any),(null as any),(null as any),View_ChartComponent_0,RenderType_ChartComponent)),
    import0.ɵdid(638976,(null as any),0,import1.ChartComponent,[import0.ElementRef],(null as any),(null as any))
  ]
  ,(ck,v) => {
    ck(v,1,0);
  },(null as any));
}
export const ChartComponentNgFactory:import0.ComponentFactory<import1.ChartComponent> = import0.ɵccf('chart',import1.ChartComponent,View_ChartComponent_Host_0,{
  type: 'type',
  data: 'data',
  options: 'options'
}
,{},([] as any[]));
